spec/expression.dd

```diff
@@ -864,13 +864,12 @@ $(GNAME CatExpression):
     $(GLINK AddExpression) $(D ~) $(GLINK MulExpression)
 )
 
-    $(P A $(I CatExpression) concatenates arrays, producing
-        a dynamic array with the result. The arrays must be
-        arrays of the same element type. If one operand is an array
-        and the other is of that array's element type, that element
-        is converted to an array of length 1 of that element,
-        and then the concatenation is performed.
-    )
+    $(P A $(I CatExpression) concatenates a container's data with other data, producing
```

Comment:
Is this actually what typical containers do? Perhaps it should be a BEST_PRACTICE

Reply:
What container doesn't it apply to? The new container doesn't necessarily have to be the same type as the LHS.